php/utils.php

```diff
@@ -1374,3 +1374,49 @@ function _proc_open_compat_win_env( $cmd, &$env ) {
 function esc_like( $text ) {
 	return addcslashes( $text, '_%\\' );
 }
+
+/**
+ * Check whether a given string is a valid JSON representation.
+ *
+ * @param string $argument       String to evaluate.
+ * @param bool   $ignore_scalars Optional. Whether to ignore scalar values.
+ *                               Defaults to true.
+ *
+ * @return bool Whether the provided string is a valid JSON representation.
+ */
+function is_json( $argument, $ignore_scalars = true ) {
+	if ( empty( $argument ) || ! is_string( $argument ) ) {
```

Comment:
To be picky this should be `if ( ! is_string( $argument ) || 0 === strlen( $argument ) ) {` to deal with if given `'0'`.

Reply:
Picky is good. Why did you merge anyway?

I'll add a bug report as a good first issue.

Reply:
See https://github.com/wp-cli/wp-cli/issues/4629

Reply:
Ha, I merged because we've so many bugs, PRs and other stuff outstanding and release is 8 days from now!